dbt/models/default/default.vw_pin_sale.sql

```diff
@@ -221,12 +221,13 @@ mydec_sales AS (
             ) AS num_single_day_sales,
             year_of_sale AS year
         FROM {{ source('sale', 'mydec') }}
+        WHERE line_2_total_parcels = 1
```

Comment:
Unfortunately, while this solves your filter issue, I believe it also drops all multisales from the mydec sample.

Reply:
In the previous view we weren't allow mydec sales into the view. We were only using them to improve dates for iasworld sales. Now that they're in the view, we need to treat them the same as iasworld sales.

Reply:
Right. It seems like in the `default.vw_pin_sale` we aren't using mydec sales to improve dates for iasworld if the iasworld sales are multi-sale - due to the `WHERE line_2_total_parcels = 1` in `default.vw_pin_sale`.

> Unfortunately, while this solves your filter issue, I believe it also drops all multisales from the mydec sample, which is something that should only happen when we set is_multisale = FALSE

If we remove the parcel filter and let in multi-sales from mydec, we will be allowing mydec date to override iasworld date for multi-sales, but this will make the following queries inconsistent, since we were not doing it for `default.vw_pin_sale`:

```sql
select count(*), is_mydec_date
from "z_ci_583_create_a_sales_view_that_combines_iasworld_mydec_and_ccrd_sales_default"."vw_pin_sale"
where source = 'iasworld'
group by is_mydec_date

select count(*), is_mydec_date
from "default"."vw_pin_sale"
group by is_mydec_date
```

Reply:
So you're right about how the view has worked in the past, but this is ultimately still something we're going to have to address. We didn't let mydec multisales in because we were only joining mydec sales to iasworld sales and that would have been a one to many join on sale doc. But now what mydec sales are in the view in their own right, we'll need to figure out a way to allow in those multisales, without joining them to iasworld sales.

I think you can probably swing this with a couple of extra conditionals in `FULL OUTER JOIN mydec_sales AS md_sales ON uq_sales.doc_no = md_sales.doc_no` to make sure multisale mydec sales don't join onto iasworld sales.

Reply:
Awesome, I'll implement this and create a second 365 filter, hopefully that way when Dan looks a this we are good to go